pkg/planner/core/issuetest/volatile_group_by_projection_test.go

```
@@ -0,0 +1,48 @@
+
+func TestVolatileGroupByProjection(t *testing.T) {
+	store := testkit.CreateMockStore(t)
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test")
+	tk.MustExec("SET @@session.tidb_allow_mpp = 0")
+	tk.MustExec("SET @@session.tidb_enable_cascades_planner = 0")
+	tk.MustExec("CREATE TABLE t (a INT, b INT, c INT, d INT)")
+	tk.MustExec("INSERT INTO t VALUES (1,1,1,10), (2,1,2,2), (3,1,1,10), (4,1,2,2)")
+
+	queries := []string{
+		"SELECT b * FLOOR(2 * RAND(177)) AS e, COUNT(d) AS cnt FROM t GROUP BY e ORDER BY e, cnt",
```

golangci-lint (2.13.2)

[warning] line-length-limit: line is 98 characters, out of limit 80

(revive)